packages/popover/popover.stories.tsx

```diff
@@ -1,85 +1,64 @@
 import * as React from "react";
-import { select } from "@storybook/addon-knobs";
+import { Story, Meta } from "@storybook/react";
+
 import { Popover } from "../";
-import { Direction } from "../dropdownable/components/Dropdownable";
 import { PrimaryDropdownButton } from "../button";
 import { Box } from "../styleUtils/modifiers";
+import { PopoverProps } from "./components/PopoverBox";
 
-const popoverStoryDecorator = storyFn => (
-  <Box textAlign="center">{storyFn()}</Box>
-);
+const directions = {
+  BottomLeft: "bottom-left",
+  BottomRight: "bottom-right",
+  BottomCenter: "bottom-center",
+  TopLeft: "top-left",
+  TopRight: "top-right",
+  TopCenter: "top-center"
+};
 
 export default {
   title: "Overlays/Popover",
-  decorators: [popoverStoryDecorator],
-  component: Popover
-};
-
-export const Default = () => (
-  <Popover
-    trigger={<PrimaryDropdownButton>open popover</PrimaryDropdownButton>}
-  >
-    <div>dropdown content</div>
-  </Popover>
-);
-
-export const WithFocusableContent = () => (
-  <Popover
-    trigger={<PrimaryDropdownButton>open popover</PrimaryDropdownButton>}
-    maxWidth={300}
-  >
-    <div>
-      <p>
-        If the dropdown is opened via keyboard navigation,{" "}
-        <a href="http://google.com/">this link</a> should get focus first, then{" "}
-        <a href="http://google.com/">this link</a> should get focus.
-      </p>
-      <p>Focus should not leave the dropdown until it is closed</p>
-    </div>
-  </Popover>
-);
+  decorators: [
+    Story => (
+      <Box textAlign="center">
+        <Story />
+      </Box>
+    )
+  ],
+  component: Popover,
+  argTypes: {
+    preferredDirections: {
```

Comment:
I'm getting a console error when selecting a preferredDirections option in these stories as well:

`Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'x')`